app/client/sagas/authSagas.js

```diff
@@ -123,7 +123,10 @@ function* saveOrgId({ payload }) {
     let query_params = parseQuery(window.location.search);
 
     // if query param and payload are matching then just reload to update navbar
-    if (query_params["org"] && payload.organisationId == query_params["org"]) {
+    if (
+      query_params["org"] &&
+      payload.organisationId.toString() === query_params["org"]
```

Comment:
higher up u parse the query param to a number and then compare, perhaps do that here too for consistency?

Reply:
fixed